packages/mcp-provider-devops/src/tools/createPullRequest.ts

```diff
@@ -87,7 +87,7 @@ export class CreatePullRequest extends McpTool<InputArgsShape, OutputArgsShape>
         return {
           content: [{
             type: "text",
-            text: `Error: Work item Name is required. Please provide a valid work item Name.`
```

Comment:
`Error: Work Item Name is required. Please provide a valid Work Item Name or a valid DevOps Center org username.`

Reply:
Just a nitpick, in other places as well, should we keep it constant as "Work Item"?